charts/fluent-bit/values.yaml

```diff
@@ -58,6 +58,16 @@ serviceMonitor:
   # selector:
   #  prometheus: my-prometheus
 
+livenessProbe:
+  # httpGet:
+  #   path: /
+  #   port: http
+
+readinessProbe:
+  # httpGet:
+  #   path: /
+  #   port: http
```

Comment:
Maybe we uncomment these and get rid of the `if/else` in the template and instead always use the toYamled value. 

Reason is at first glance, one might think none is configured by default, and also it will allow the user to remove probes if they so wish to.

Reply:
Good idea. Updated.